### PR TITLE
Add GPG auto-sign commands

### PR DESCRIPTION
**Pull request by sih4sing5hong5**
### Why:

Add automatic signing commands in docs.

Relative issues:

- https://github.com/github/docs/issues/18218#issuecomment-1137336102
- https://github.com/github/docs/pull/15311

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Thank for amazing [preview](https://docs-21497-3023b2.preview.ghdocs.com/en/enterprise-server@3.6/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key?platform=windows)

This commands is in sixth steps:

screen shot

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by sih4sing5hong5** [comment]
It seems Link Checker was broken.
```
Run node .github/actions/rendered-content-link-checker.js
  node .github/actions/rendered-content-link-checker.js
  shell: /usr/bin/bash -e {0}
  env:
    LEVEL: critical
    ACTION_RUN_URL: https://github.com/github/docs/actions/runs/3286389283
    GITHUB_TOKEN: 
    SHOULD_COMMENT: true
    CHECK_EXTERNAL_LINKS: false
    CREATE_REPORT: false
file:///home/runner/work/docs/docs/.github/actions/lib/get-env-inputs.js:13
        throw new Error(`You must supply a ${envVarName} environment variable`)
              ^

Error: You must supply a GITHUB_TOKEN environment variable
    at file:///home/runner/work/docs/docs/.github/actions/lib/get-env-inputs.js:13:15
    at Array.map (<anonymous>)
    at getEnvInputs (file:///home/runner/work/docs/docs/.github/actions/lib/get-env-inputs.js:10:13)
    at file:///home/runner/work/docs/docs/.github/actions/rendered-content-link-checker.js:55:3
Error: Process completed with exit code 1.
```

**Comment by cmwilson21** [comment]
@sih4sing5hong5 Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/gpg/set-auto-sign.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1. To set automatic GPG signing in Git, paste the text below:
+   ```shell
+   $ git config --global commit.gpgSign true
+   ```
```
Comment (isaacmbrown) [inline data/reusables/gpg/set-auto-sign.md:4]:
```suggestion
1. Optionally, to configure Git to sign all commits by default, enter the following command:

   ```Shell
   $ git config --global commit.gpgSign true
   ```
   
   For more information, see 
```

**Review by isaacmbrown — COMMENTED** [review]
Hi @sih4sing5hong5, thanks for raising the PR ✨  This looks like a worthwhile addition. Have just added a little bit of information to the step to add some clarity 🙂

**Review by isaacmbrown — APPROVED** [review]

**Comment by sih4sing5hong5** [comment]
> Hi @sih4sing5hong5, thanks for raising the PR sparkles This looks like a worthwhile addition. Have just added a little bit of information to the step to add some clarity slightly_smiling_face

@isaacmbrown Thanks. It's more user-friendly!! :+1:

**Pull request merged**